tools/influx_si/influx_si.xml

```diff
@@ -0,0 +1,85 @@
+<tool id="influx_si" name="influx_si" version="0.9.0+galaxy0" python_template_version="3.5">
```

Comment:
Why not using `@TOOL_VERSION@` instead of `0.9.0` ?
What mean `5.1.0` and `0.9.0` ?

Reply:
@TOOL_VERSION@ (currently 5.1.0) was used for the version of the influx_si which can be used independently of galaxy. While 0.9.0 (now 0.9.1) is the version of the xml wrapper. That's why it would not be helpful to use one instead of another.
As I removed macro.xml, the token @TOOL_VERSION@ is not used anymore.

Reply:
@lecorguille I have misunderstood you point when you said:
> Une nouvelle convention de nommage recommande : 0.6.0+galaxy0 pour gérer indépendamment le version de l'outil sous-jacent et éventuellement les versions de wrappers pour la même version d'outil.

Now, I set the version to "@TOOL_VERSION@+galaxy0".